bundles/org.openhab.binding.myenergi/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,455 @@
+			<channel id="clamp-power5" typeId="clamp-power"/>
+			<channel id="clamp-name6" typeId="clamp-name"/>
+			<channel id="clamp-power6" typeId="clamp-power"/>
+		</channels>
+
+		<properties>
+			<property name="serialNumber"/>
+			<property name="firmwareVersion"/>
+			<property name="numberOfPhases"/> <!-- The number of phases in this installation -->
+		</properties>
+
+		<config-description>
+			<parameter name="refreshInterval" type="integer" required="false" min="5">
+				<label>Refresh Interval</label>
+				<description>Query interval (in seconds) for device measurements (generation, consumption etc.)</description>
+				<default>30</default>
+			</parameter>
+		</config-description>
+
+	</thing-type>
+
+	<thing-type id="eddi">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="bridge"/>
+		</supported-bridge-type-refs>
+
+		<label>Eddi</label>
+		<description>myenergi solar power diverter</description>
+
+		<channels>
+			<channel id="last-updated-time" typeId="last-updated-time"/>
+			<channel id="supply-voltage" typeId="voltage"/>
+			<channel id="supply-frequency" typeId="frequency"/>
+			<channel id="diverter-status" typeId="diverter-status"/>
+			<channel id="diverter-priority" typeId="diverter-priority"/>
+			<channel id="phase" typeId="phase"/>
+			<channel id="grid-power" typeId="grid-power"/>
+			<channel id="boost-mode" typeId="boost-mode"/>
+			<channel id="energy-transferred" typeId="energy-transferred"/>
+			<channel id="generated-power" typeId="generated-power"/>
+			<channel id="diverted-power" typeId="diverted-power"/>
+			<channel id="boost-remaining" typeId="boost-remaining"/>
+			<channel id="active-heater" typeId="active-heater"/>
+			<channel id="heater-priority" typeId="heater-priority"/>
+			<channel id="heater-name1" typeId="heater-name1"/>
+			<channel id="heater-name2" typeId="heater-name2"/>
+			<channel id="temperature1" typeId="temperature1"/>
+			<channel id="temperature2" typeId="temperature2"/>
+			<channel id="clamp-name1" typeId="clamp-name"/>
+			<channel id="clamp-power1" typeId="clamp-power"/>
+			<channel id="clamp-name2" typeId="clamp-name"/>
+			<channel id="clamp-power2" typeId="clamp-power"/>
+			<channel id="clamp-name3" typeId="clamp-name"/>
+			<channel id="clamp-power3" typeId="clamp-power"/>
+		</channels>
+
+		<properties>
+			<property name="serialNumber"/>
+			<property name="firmwareVersion"/>
+		</properties>
+
+		<config-description>
+			<parameter name="refreshInterval" type="integer" required="false" min="5">
+				<label>Refresh Interval</label>
+				<description>Query interval (in seconds) for device measurements (generation, consumption etc.)</description>
+				<default>30</default>
+			</parameter>
+		</config-description>
+
+	</thing-type>
+
+	<channel-type id="last-updated-time">
+		<item-type>DateTime</item-type>
+		<label>Last Updated Time</label>
+		<description>The time when the device was last updated</description>
+		<state readOnly="true" pattern="%1$tF %1$tR"/>
```

Comment:
Can you add semantic tags (point and optional property) to all channel type's?

For reference: 
https://www.openhab.org/docs/developer/bindings/thing-xml.html#default-point-and-property-tags